delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -63,11 +63,17 @@ class ResourcesRoutesSpec extends BaseRouteSpec with IOFromMap {
   private val schema1      = SchemaGen.schema(nxv + "myschema", project.value.ref, schemaSource.removeKeys(keywords.id))
   private val schema2      = SchemaGen.schema(schema.Person, project.value.ref, schemaSource.removeKeys(keywords.id))
 
-  private val myId                        = nxv + "myid"  // Resource created against no schema with id present on the payload
-  private val myId2                       = nxv + "myid2" // Resource created against schema1 with id present on the payload
-  private val myId3                       = nxv + "myid3" // Resource created against no schema with id passed and present on the payload
-  private val myId4                       = nxv + "myid4" // Resource created against schema1 with id passed and present on the payload
-  private val myId5                       = nxv + "myid5" // Resource created against schema1 with id passed and present on the payload
+  private val myId  = nxv + "myid"  // Resource created against no schema with id present on the payload
+  private val myId2 = nxv + "myid2" // Resource created against schema1 with id present on the payload
+  private val myId3 = nxv + "myid3" // Resource created against no schema with id passed and present on the payload
+  private val myId4 = nxv + "myid4" // Resource created against schema1 with id passed and present on the payload
+  private val myId5 = nxv + "myid5" // Resource created against schema1 with id passed and present on the payload
+  private val myId6 = nxv + "myid6" // Resource created and tagged against no schema with id present on the payload
+  private val myId7 = nxv + "myid7" // Resource created and tagged against no schema with id present on the payload
+  private val myId8 =
+    nxv + "myid8" // Resource created and tagged against no schema with id passed and present on the payload
+  private val myId9 =
+    nxv + "myid9" // Resource created and tagged against schema1 with id passed and present on the payload
```

Comment:
This is kinda gross, I'd like to refactor it at some point to use generated data

Reply:
Yep or break it down in some way, this does too many things